packages/icrc-ledger-types/src/icrc1/account.rs

```diff
@@ -167,14 +170,82 @@ impl FromStr for Account {
     }
 }
 
+const MAX_SERIALIZATION_LEN: u32 = 62;
+
+impl Storable for Account {
```

Comment:
Wouldn't it be more robust to use a standard serialization/deserialization library (e.g., cbor/candid)?

Reply:
I did a test with candid and the maximum serialized size went up from 62 to 90 bytes. I suspect we will see a similar increase for serde as it also needs to store some metadata. OTOH, the code is indeed much simpler and I see a lot of other implementations use it. Or we ok with this memory increase? cc @mbjorkqvist

Reply:
I tested current manual serialization/deserialization and minicbor. Below are results for serialization+deserialization of maximum size Account.
Instructions manual: 4365
Instructions minicbor: 4671
Size manual: 62
Size minicbor: 66